beastify/popup/choose_beast.html

```diff
@@ -8,10 +8,10 @@
 
 <body>
   <div id="popup-content">
-    <button>Frog</button>
-    <button>Turtle</button>
-    <button>Snake</button>
-    <button type="reset">Reset</button>
+    <button class="button beast">Frog</button>
```

Comment:
this shouldn't be necessary. Just style based on `<button>` and `[type="reset"]`